components/script/Cargo.toml

```diff
@@ -68,7 +68,7 @@ metrics = {path = "../metrics"}
 mitochondria = "1.1.2"
 mime = "0.2.1"
 mime_guess = "1.8.0"
-mozjs = "0.9.0"
+mozjs = { git = "https://github.com/CYBAI/rust-mozjs.git", branch = "getpromiseishandled" }
```

Comment:
The `mozjs` version will be updated once servo/rust-mozjs#443 reviewed, merged and the package deployed!